Bundle/UKCore-Bundle-MichaelJonesRequest-Example_minimal.json

```diff
@@ -125,44 +125,42 @@
 				"status": "active",
 				"intent": "order",
 				"category": [
-					{ 
+					{
```

Comment:
Can you remove this completely? The malignant neoplasm code is not a category, and we already have a code for the servicerequest.